sites/main-site/src/pages/index.astro

```diff
@@ -100,9 +101,14 @@ import AnnouncementBanner from '@components/navbar/AnnouncementBanner.astro';
                 </HomepageCard>
             </div>
             <div class="col">
-                <HomepageCard title="Stable Releases" icon="line-md:check-list-3-twotone" icon_class="text-success">
-                    nf-core pipelines use GitHub releases to tag stable versions of the code and software, making
-                    pipeline runs totally reproducible.
+                <HomepageCard
+                    title="Open source"
+                    name="opensource"
+                    icon="cib:open-source-initiative"
+                    icon_class="text-success"
+                >
+                    All nf-core code is licensed under the MIT license and is available on GitHub. Pipelines can be used
+                    as they are or forked and modified as needed.
```

Comment:
```suggestion
                    All nf-core code is licensed under the MIT license and is available on GitHub. Pipelines can be used
                    modified, forked, by anyone.
```

Reply:
Feel like we need an `and` there somewhere.. 🤔 

```suggestion
                    All nf-core code is licensed under the MIT license and is available on GitHub. Pipelines can be used
                    modified and forked, by anyone.
```